test/10-ctrl.bats

```diff
@@ -0,0 +1,118 @@
+#!/usr/bin/env bats
+
+load test_helper
+
+setup() {
+    check_conmon_binary
+    check_runtime_binary
+    setup_container_env "while [ ! -f /tmp/test.txt ]; do /busybox sleep 0.1; done; /busybox stty size" "true"
+    generate_process_spec "echo 'Hello from exec!' && echo 'Hello there!' > /tmp/test.txt"
+}
+
+teardown() {
+    cleanup_test_env
+}
+
+# Helper function to start conmon, send the control command to it and terminate it.
+test_ctl_command() {
+    local command="$1"
+    start_conmon_with_default_args --log-path "k8s-file:$LOG_PATH" -t
+    wait_for_runtime_status "$CTR_ID" running
+
+    echo "$command" > ${CTL_PATH}
+
+    start_conmon_with_default_args \
+        --log-path "k8s-file:$LOG_PATH.exec" \
+        --exec \
+        --exec-process-spec "${BUNDLE_PATH}/process.json"
+
+    wait_for_runtime_status "$CTR_ID" stopped
+}
+
+# Helper function to send the resize command. Fails if the resize command
+# triggers the tty resize.
+test_resize_command_fail() {
+    local command="$1"
+    test_ctl_command "$command"
+
+    # Check that the main process noticed the /tmp/test.txt.
+    assert_file_exists "$LOG_PATH"
+    run cat "$LOG_PATH"
+    # No terminal resize
+    assert "${output}" =~ "standard input"
+}
+
+# Helper function to send the resize command. Fails if the resize command
+# does not trigger the tty resize.
+test_resize_command_ok() {
+    local command="$1"
+    test_ctl_command "$command"
+
+    # Check that the main process noticed the /tmp/test.txt.
+    assert_file_exists "$LOG_PATH"
+    run cat "$LOG_PATH"
+    assert "${output}" =~ "2 2"
```

Comment:
This is wrong? `"2 2"` is hardcoded, shouldn't `test_resize_command_ok()` be invoked with `$1` - command and `$2` - expected size? E.g.:

```
@test "ctrl: resize with leading zeros" {
    test_resize_command_ok "1 0010 0020" "10 20"
}
```